### ISSUE TITLE
PRODUCTION (CONVENINGS): Connect leaders to leaders

### ISSUE THREAD

**Issue by ldecoursy** [MozillaFoundation/Goals#17]
## GOALS
1. Storytelling app MVP
2. Event App 2.0

**Comment by ldecoursy** [comment]
## MONTH
- ACCOMPLISHMENTS (What progress did we make towards these goals?)
- LEARNINGS (What challenges did we face/what went well/what did we learn?)
- PRIORITIES FOR NEXT MONTH (What are we focusing on next?)
- ON TRACK? BLOCKED? (List dependencies, or collaborations with other teams at MoFo/MoCo)

**Comment by ldecoursy** [comment]
@xmatthewx When you're back, can you help update this ticket for July/Aug using the template in the comment above? thanks!

**Comment by ldecoursy** [comment]
ping @xmatthewx when you can!

**Comment by ldecoursy** [comment]
@xmatthewx Can you please update here against these goals using the template above - maybe one comment with a Q3 update rather than separate comments for each month?

**Comment by xmatthewx** [comment]
## Q3
- ACCOMPLISHMENTS (What progress did we make towards these goals?)
  - We launched Network Pulse, our v1 prototype, and started a pilot test with program staff. This iteration includes the ability search and favorite assets, and browse them by our five key issues.  https://mzl.la/pulse
  - We iterated on the event app with improved UI and UX for use at MozFest. It includes refined navigation and capacity for multi-lingual session descriptions. https://mozilla.github.io/mozfest-event-app/
  - We designed a strong theme and compelling visual identity for this year's MozFest, and began production of print materials and space design.
- LEARNINGS (What challenges did we face/what went well/what did we learn?)
  - With Pulse, we surveyed program staff about their operational challenges and learned to think about staff as users, as the nearest nodes in our network of members. 
  - For MozFest, we leveraged London-based consultants for some design of our space and print materials. We learned many things about the benefits and challenges of leveraging contractors for critical production.  
- PRIORITIES FOR NEXT MONTH (What are we focusing on next?)
  - We will prepare the use of Pulse at MozFest to extend the science fair throughout the weekend and capture activity at the festival. We will support program staff during their pilot use of this tool.
  - We wrap up production of print and digital assets for MozFest, while supporting the release of the festival's schedule. We will continue updates to the festivals website.
- ON TRACK? BLOCKED? (List dependencies, or collaborations with other teams at MoFo/MoCo)
  - On track. The complexity of MozFest will require more focus and more hands throughout October.

I'd like to add a few words about the Mentorship program. I'll edit this comment when I've gathered that.

**Comment by abbycabs** [comment]
@xmatthewx I filled out the template for mentorship program stuff. Feel free to edit/merge with your comment!

GOAL: launch mentorship program
- Strengthen network connections
- Increase & sustain project-based engagement at MozFest 2016

Q3 - Mentorship Program
- ACCOMPLISHMENTS (What progress did we make towards these goals?)
  - Launched Mentorship Program
  - Gathered nominations from across MoFo & Participation of community members bringing open projects to MozFest 2016. Offered 22 open project leads a spot in the mentorship program.
  - Selected, offered, and onboarded 11 mentors from our previous mentorship round
  - Gathered an additional 19 experts willing to provide 1:1 consultation on projects during the program
- LEARNINGS (What challenges did we face/what went well/what did we learn?)
  - Difficult to get a diverse nominations across MoFo. Might be easier next time with more awareness of the program.
  - Ongoing feedback on the draft version of the OLTS
- PRIORITIES FOR NEXT MONTH (What are we focusing on next?)
  - finishing this round and prepping participants for MozFest
  - continue testing the current state of the OLTS and incorporating feedback
  - offering a self-direction version of this training to all MozFest facilitators
- ON TRACK? BLOCKED? (List dependencies, or collaborations with other teams at MoFo/MoCo)
  - on track

**Comment by ldecoursy** [comment]
ping @xmatthewx for Oct updates.  

@acabunoc I wonder if your Sept update (and hopefully an Oct update too) belongs in #12 instead

**Comment by xmatthewx** [comment]
## October 2016

Lots of accomplishments and measurable wins.

### Schedule App

We made small improvements to the UI and UX of the schedule app. This minimal investment of time and energy paid off in big ways.
-  Attendees were able to navigate the complex schedule. **Sessions were full, reportedly higher than ever before**, despite the sheer number of options.
-  We captured a ton of data on the preferences of attendees. **This data unlocks new knowledge about our community** that can inform not only the next festival but decisions across all of our programs. 
  - **Session details opened 40,000 times by more than 10,000 unique people.** 6,400 people outside the UK reading schedule session details, more than 2,000 remote during the weekend? Pretty cool. (This is true engagement, not simply visiting a website.)
  - Aggregate Session data (views and favs) gives us unique **insight on how people engage with our topics**. This can impact our programs and should also help plan spaces and pathways next year. [Spreadsheet](https://docs.google.com/spreadsheets/d/1sXEXiAV58ZA4qJtOAw546-dBjGkbTFheA_-klQglT2M/edit) and [github issue](https://github.com/MozillaFoundation/Mozfest2016_production/issues/220).
- [App feedback](https://github.com/mozilla/mozfest-event-app/issues/484)

### Pulse
- Our internal pilot with program staff is going well. Steady progress building participation.
- Pulse captured 3 dozen high value projects connected to MozFest, giving us strong potential to extend the festival, amplify these projects, and foster collaboration. **MozFest projects live on in [Pulse](https://mzl.la/pulse) and on [Mozfest site](https://mozillafestival.org/projects)**
- **Pulse gained momentum and support among staff, connecting it with MoCo marketing, design, and innovation endeavors.**
- Post-Mozfest, we're seeing more unsolicited submissions, including a few from MoCo and community. This was not a goal, but is a useful indicator. 
- Pulse captures metadata on our Issue area. Too early for conclusions, but there some **interesting signals** on where community and staff endeavors fall along these vectors. 

### MozFest Design

Wayfinding, Festival guide, Website and other design elements were aesthetically beautiful and deeply impacted the experience for staff, facilitators and attendees. 
- Design helped the fest operate more smoothly
- Design contributed toward a renewed spirit that was palpable. Staff and community were excited and proud.
  - **Fun evidence**: The community leveraged the design to express themselves and celebrate their experience. They shared countless photos of localized posters and screenshots of their session in the schedule app on social media. Design helped them tell their story, which helped us spread our story. [WIP blogpost](https://docs.google.com/document/d/1AZLyI5ma8mW19RG0lrlJHCDDAW_4_TStG5tN0CP6Pm8/edit)
  - Design was well coordinated with the fine art, 5 issue installations, and other key elements of the fest.
  - We leveraged consultants in new ways to increase the quality of design and production while minimizing the stress on staff for a sustainable endeavor.
  -  Despite the complex production and installation requirements, everything went according to plan. 

## LEARNINGS

### MozFest Design
- We have captured detailed lessons about each item produced for MozFest. This will greatly simplify design and production for next year. [Tracked here](https://github.com/MozillaFoundation/Mozfest2016_production/issues/222).

### Pulse
- Pulse resonates with community and staff, including CEL, MoCo Marketing, Mitchell. 
  It offers a potential answer to the imperative need to make our Network’s footprint visible.  

### Event app
- Pre-MozFest, there was internal debate on whether to invest further in the Event app or switch to an off the shelf solution. The late hour debate was too late to shift to a new app, but led us to not invest to improving the app's underlying technology. The rationale was good, but the timing was bad. As a result, we then wasted a significant amount of Developer staff time supporting schedule content input and edits. It led to a great festival, but that time is otherwise sunk cost.
- People love the app.
- The app works great, even with intermittent internet connectivity.
- The app is a tremendous resource for gathering insight on our community, their interests, and the words we use to describe our issues.
- We need to learn from other peer un/conference planners whether they have a go to tool, or not. Then make an informed decision on this app. 

## PRIORITIES FOR NEXT MONTH

### MozFest
- Collating documentation on the fest experience [WIP content for Medium](https://docs.google.com/document/d/1AZLyI5ma8mW19RG0lrlJHCDDAW_4_TStG5tN0CP6Pm8/edit). 
- Capturing lessons learned.

### Pulse
- Plan a rebuild, from v1 Prototype to v2 Production
- Assess pilot with Program staff
- Prep for Hawaii to explore integration with MoCo endeavors  

## ON TRACK? BLOCKED?
- On track
- Slightly blocked on engineering capacity, mostly due to PTO and travel

**Comment by ldecoursy** [comment]
@xmatthewx Anything to add here for Nov/Dec?
